client/hosting/sites/components/style.scss

```diff
@@ -1118,16 +1118,20 @@
 	}
 }
 
-/* See wp-calypso/client/layout/style.scss for the original style - this is removed on sites dashboard to enable pagination to display properly. */
-.layout__primary .preview-hidden {
-	@include breakpoint-deprecated( ">960px" ) {
-		padding-bottom: 0 !important;
+// Styles need to ensure they do not bleed into other areas of calypso once loaded.
+// If we only apply to layout__primary, we will have bleed going from the dashboard to site level calypso.
+.is-global-sidebar-visible {
```

Comment:
This is the most accurate higher level selector I can find for these cases. it seems to be present everywhere we need these styles to take effect.

Unfortunately using section selectors doesn't work: while the sites-dashboard section isnt used outside of the sites dashboard, the sites section that is used at the overview page is the same section used in the rest of calypso like my-home. So instead I opted for the global sidebar selector check.

Reply:
Edit - I added more selectors to make sure this only appears at dashboard and overview sections, and not into other global areas like reader and me.